README.md

```diff
@@ -1,21 +1,66 @@
 # LazyBroadcast.jl
 
-A package for constructing Broadcasted objects from broadcast expressions.
+LazyBroadcast.jl provides a macro, `LazyBroadcast.@lazy_broadcasted` to
+transform a given Julia broadcast expression into a
+`Base.Broadcast.Broadcasted` object, without materializing it.
 
-Our test suite has a simple example demonstrating its use:
+For more information about Julia broadcasting, please see https://docs.julialang.org/en/v1/manual/arrays/#Broadcasting.
+
+This utility is useful in a few situations:
+
+ - Debugging broadcast machinery
+ - Fusing operations in multiple broadcast expressions
```

Comment:
it would be nice to include an example of how `@lazy_broadcasted` can be used to do this

Reply:
It's a bit difficult to do this because an example requires implementation details, and it's a fair bit of work to demonstrate. This is basically what MultiBroadcastFusion.jl attempts to do, maybe we can point to the MultiBroadcastFusion repo?

Reply:
I added a reference to MultiBroadcastFusion.jl, so that users can see examples there.